website/src/components/Layout.tsx

```diff
@@ -51,4 +51,22 @@ export const getDashboardLayout = (page: React.ReactElement) => (
   </div>
 );
 
+export const getAdminLayout = (page: React.ReactElement) => (
+  <div className="grid grid-rows-[min-content_1fr_min-content] h-full justify-items-stretch">
+    <Header transparent={true} />
+    <SideMenuLayout
+      menuButtonOptions={[
+        {
+          label: "Users",
+          pathname: "/admin",
+          desc: "Users Dashboard",
+          icon: FiUsers,
+        },
+      ]}
+    >
```

Comment:
I think the admin should also be able to see the dashboard & messages, at least for development purposes.

Reply:
I can add the links as in the original dashboard layout
![image](https://user-images.githubusercontent.com/30481105/211151447-70aaacb5-d58a-4c94-9c7c-93d7dea0c131.png)

But this would present another problem. Now we are not providing layout based on the user's role. It's basically a whole another layout component for `/admin` route. 
So when admin user interacts with these links to `/dashboard` and `/messages` sidebar won't include admin's menu options.

Reply:
ah, I see.. that is actually a problem.

would it make sense to add the items in the original dashboard if the user is admin?

Reply:
I think it would be reasonable. But need some comments from @fozziethebeat as the original issue mentioned separate layouts.

Reply:
In my mind the admin page would be a separate view that's accessible from the user drop down in the header.  It would have the same look and feel of the contributing dashboard but with different side menu options.

That way, its *clearly* a different mode of interaction.  This is why I was thinking we'd use the same set of generic components but fill them independently.

This PR matches what I was requesting.

Reply:
I'm going to merge this since it matched the design I had in mind (but maybe didnt' explain super clearly).  Let's discuss this in the next live call to see how we might want to change the design.